File: 2310817120009_AlysaArmelia_PemrogramanI_Modul3/Python/PRAK304-2310817120009-AlysaArmelia.py
bilangan = int(input())

if 1 <= bilangan <= 10:
    print("Satuan")

elif bilangan == 0:
    print("Nol")

elif bilangan >= 100:
    print("Anda Mengimput Melebihi Limit Bilangan")
    
elif 20 <= bilangan <= 99:
    print("Puluhan")
    
else:
    print("Belasan")

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul3/Bahasa C/PRAK304-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main()
{

int bilangan;

printf("");
scanf("%d", &bilangan);

if (bilangan >= 1 && bilangan <= 10){
    printf("Satuan");
    
    }   else if (bilangan == 0){
    printf("Nol");
    
    }   else if (bilangan >= 100){
    printf("Anda Mengimput Melebihi Limit Bilangan");
    
    }   else if (bilangan >= 20 && bilangan <= 99){
    printf("Puluhan");
    
    }   else {
    printf("Belasan");
}
return 0;
}
